### ISSUE TITLE
mesh segmentation - parameters for training

### ISSUE THREAD

**Issue by jajakob** [tensorflow/graphics#30]
Hi guys,
I'm curious what parameters you guys have used for training the model for the mesh segmentation demo. The ones that would be great to know, are:
- learning rate (0.001 like FeaStNet?), beta1 and epsilon for the Adam optimiser
- number of training epochs (and number of steps per epoch) or number of iterations
- if used, after how many epochs did you evaluate?
- train io params ? E.g.:
```
train_io_params = {
    'sloppy': False,
    'shuffle': True,
    'repeat': True,
}
```
Best,
Jakob

**Comment by avneesh-sud** [comment]
Hi Jakob,

Here are the parameters for training:
total_steps = 150k
batch_size = 8
num_filters (for FeaStNet) = 8
encoder_filter_dims = [32, 64, 128]
learning_rate = 0.001, with exponential decay and following parameters: learning_rate_decay_steps = 10000, learning_rate_decay_rate = 0.95
adam_beta = 0.9
adam_epsilon = 1e-8
